Tools/Scripts/libraries/webkitbugspy/webkitbugspy/mocks/radar.py

```diff
@@ -122,7 +122,7 @@ def __init__(self, name):
 
     class Tentpole(object):
         def __init__(self, name):
-            self.name = name
+            self.name = name['name'] if isinstance(name, dict) else name
```

Comment:
I assume this is because the data representation for this object is a dictionary for the API request. When would it be a string or otherwise?

Reply:
I was trying to not change the existing API, but it's only used from within `webkitbugspy` for testing.